as/json.js

```diff
@@ -192,9 +194,11 @@ TChannelJSON.prototype.register = function register(
         }
 
         var v = parseResult.value;
+        req.channel.startSpan(req, v.head);
         handlerFunc(opts, req, v.head, v.body, onResponse);
 
         function onResponse(err, respObject) {
+            req.channel.finishSpan(req.openSpan, err);
```

Comment:
nit: if it was me, I would keep this symmetric with startSpan and pass `(req, err)`, without revealing there's such a thing as `openSpan`